include/APIClient/GPU.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <restclient-cpp/restclient.h>
+#include <restclient-cpp/connection.h>
+
+#include <APIClient/Common.hpp>
+#include <APIClient/Router.hpp>
+
+namespace APIClient {
+
+namespace GPU {
+
+    using ResponseCode = int;
+    using JSONResponse = std::tuple<ResponseCode, nlohmann::json>;
+
+    JSONResponse put(nlohmann::json vulkanInfosJson);
+    JSONResponse getAll();
+    JSONResponse get(std::string id);
+};
```

Comment:
Don't put a `;` after a namespace declaration.
Put a comment with the name of the namespace.